strapi/design-system
# [fix] Remove component with unnecessary computeScrollIntoView

### What does it do?

- Removes a call to [maintainScrollVisibility](https://github.com/strapi/design-system/blob/803fbfde0bd7049a9a24dbf08a7a311ef9513e4a/packages/strapi-design-system/src/Combobox/utils.js#L92-L93) which is causing a bug when no option is selected.

### Why is it needed?

- When the combobox is in a container with a scrollable height clicking the combobox input causes a scroll down the page

### How to test it?
- Do the following on main to see the bug then on this branch to see the fix
- In the Combobox story, wrap the Combobox in a container with a massive height like 
```
  <div style={{height: '200vh'}}><Combobox/></div>
```
- Click the input to see the options, it should not scroll down the page.

### Related issue(s)/PR(s)

resolves: https://github.com/strapi/design-system/issues/818

https://github.com/strapi/strapi/pull/15504
